stdlib/mexpr/pprint.mc

```diff
@@ -1,43 +1,60 @@
--- TODO Update and organize according to changes in ast.mc
 
 include "char.mc"
 include "option.mc"
 include "seq.mc"
 include "string.mc"
+include "name.mc"
 
-include "ast.mc"
-include "ast-builder.mc"
+include "mexpr/ast.mc"
 
 let spacing = lam indent. makeSeq indent ' '
 let newline = lam indent. concat "\n" (spacing indent)
 
 -- Set spacing on increment
-let incr = lam indent. addi indent 4
+let incr = lam indent. addi indent 2
+
+let symbolDelim = "'"
+
+-- 'sym2string sym' returns a string corresponding to the symbol 'sym'. This
+-- exposes the underlying symbol representation, but is necessary for pretty
+-- printing.
+let sym2string = lam s. int2string (sym2int s)
```

Comment:
Should this perhaps be part of a more general `symbol.mc` library instead? Sounds like something that could be useful for other debugging purposes.

Reply:
Sure, I can do that.